packages/client/ts/controllers/info.ts

```diff
@@ -21,18 +21,18 @@ export default class InfoController {
         this.destroyQueue = new Queue();
     }
 
-    create(type: number, data: any[], x: number, y: number): void {
+    create(type: number, data: unknown[], x: number, y: number): void {
```

Comment:
I guess `data` is of `number | string` type, based on what is in line 29: `const damage = data.shift() as number | string;`

It would be nice if that was `number` or `string`. We would avoid type
casting like:

`const dId = this.generateId(this.game.time, damage as number, x, y);`